<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientGUIClass</class>
 <widget class="QMainWindow" name="ClientGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget_comptes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>No Compte</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type de compte</string>
     </property>
    </column>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_infoComptes">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>361</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_infoClient">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <widget class="QMenu" name="menuAjouter_un_compte">
     <property name="title">
      <string>Ajouter un compte</string>
     </property>
     <addaction name="actionCheque"/>
     <addaction name="actionEpargne"/>
    </widget>
    <addaction name="menuAjouter_un_compte"/>
    <addaction name="actionSupprimer_un_compte"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionSupprimer_un_compte">
   <property name="text">
    <string>Supprimer un compte</string>
   </property>
  </action>
  <action name="actionCompte_Ch_que">
   <property name="text">
    <string>Compte Cheque</string>
   </property>
  </action>
  <action name="actionCompte_pargne">
   <property name="text">
    <string>Compte Epargne</string>
   </property>
  </action>
  <action name="actionCheque">
   <property name="text">
    <string>Compte Chèque</string>
   </property>
  </action>
  <action name="actionEpargne">
   <property name="text">
    <string>Compte Épargne</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>ClientGUIClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
